generated/nidaqmx/_base_interpreter.py

```diff
@@ -0,0 +1,1621 @@
+# Do not edit this file; it was automatically generated.
+
+class BaseInterpreter(object):
+    """
+    Contains signature of functions for all DAQmx APIs.
+    """
+    def add_cdaq_sync_connection(self, port_list):
+        pass
+
+    def add_global_chans_to_task(self, task, channel_names):
```

Comment:
Does this mean that the interpreter isn't going to store the task handle as previously discussed? That might be okay. The type of `task` would be interpreter-dependent, but it would be stored in the `Task` object, not the interpreter object. 

If `BaseInterpreter` had type annotations, it should use `task: object` not `task: Any`: https://mypy.readthedocs.io/en/latest/dynamic_typing.html#any-vs-object

Reply:
Yes. Interpreter will not have functions like 'set_task_handle'.